google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/package-info.java

```diff
@@ -18,30 +18,26 @@
  * A client for BigQuery – A fully managed, petabyte scale, low cost enterprise data warehouse for
  * analytics.
  *
- * <p>A simple usage example showing how to create a table if it does not exist and load data into
- * it. For the complete source code see <a
- * href="https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/CreateTableAndLoadData.java">
- * CreateTableAndLoadData.java</a>.
+ * <p>A simple usage example showing how to create a table in Bigquery. For the complete source code see <a
+ * href="https://github.com/googleapis/java-bigquery/blob/main/samples/snippets/src/main/java/com/example/bigquery/CreateTable.java">
```

Comment:
This exists https://github.com/googleapis/java-bigquery/blob/main/samples/snippets/src/main/java/com/example/bigquery/CreateTable.java. Good.